src/resources/views/crud/columns/markdown.blade.php

```diff
@@ -1,6 +1,6 @@
 @php
     $column['text'] = Illuminate\Mail\Markdown::parse($entry->{$column['name']} ?? '');
-    $column['escaped'] = $column['escaped'] ?? false;
+    $column['escaped'] = $column['escaped'] ?? true;
```

Comment:
I think the current implementation is confusing. There should probably be a `strip_tags` attribute too, so that people choose whether to do that.

Reply:
Ok so I've decided it's a bad idea to add `strip_tags` functionality to this column. Why? Because it's a convenience function - if it makes sense to do, it should't be done here, it should be done in the Model, so that _any time_ you get that attribute `strip_tags` is run. By doing it here, we'd make it convenient for the dev.

But... if you add it to this column, then... you might also want it inside `textarea`. Or `text`. Or... any other column, really. Where do you stop?

So instead of increasing the security for this column type, I've decided to keep it as it is, and default to `escaped => false`.